src/components/Label/Label.stories.tsx

```diff
@@ -195,5 +195,16 @@ stories
       customColor={(text("customColor", "#B267B2"))}
       outlined={boolean("Outlined", false)}
       focus={boolean("Focus", false)}
-    />
-  ))
+    />))
+    .add("should respect genric DOM attributes", () => (
+      <Label
+        text={text("text", "Label")}
+        expanded={boolean("Expanded", false)}
+        size={select("Size", sizeOptions, "medium")}
+        colorPreset={select("Color preset", colorThemeOptions, "neutral")}
+        rounded={boolean("Rounded", false)}
+        outlined={boolean("Outlined", false)}
+        focus={boolean("Focus", false)}
+        onHover={() => console.log("hovered...")}
```

Comment:
This should be changed to a valid event like `onMouseOver` or `onClick`, and we can use the `action` addon of storybook to register the action instead of console log.